src/main/java/org/broadinstitute/dsm/db/dto/ddp/kitrequest/KitRequestDto.java

```diff
@@ -59,4 +60,8 @@ public KitRequestDto(
         this.uploadReason = uploadReason;
         this.orderTransmitted_at = orderTransmitted_at;
     }
+
+    public boolean hasUploadReason() {
+        return StringUtils.isBlank(uploadReason);
```

Comment:
Don't we want the negation here?
```suggestion
        return StringUtils.isNotBlank(uploadReason);
```